index.js

```diff
@@ -8,6 +8,17 @@ const notes = []
 
 app.use(jsonParser)
 
+app.put('/notes/:id', (req, res) => {
+  const noteId = parseInt(req.params.id, 10)
```

Comment:
What if you decide to define notes with more properties? Can you generalize this solution?